mhs-reference-implementation/selenium_tests/page_objects/methods.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from selenium_tests.page_objects import interactions, xml_parser
+
+HUMAN_READABLE = 'Payload stuff'
+
+
+def __init__(self, driver):
+    self.driver = driver
+
+
+def get_asid():
+    ###The asid should be stored in data\local_asid file (which is excluded from GIT
+    with open(os.path.join(os.getcwd(), "selenium_tests\\data\\local_asid")) as asid_file:
+        asid = asid_file.readline()
+    asid_file.close()
+
```

Comment:
There is a method in the `FileUtilities` class that will handle this file reading, might be worth extracting the file path into a constant. Previously we using the `Path` library to build paths and there is also a `ROOT_DIR` constant in the `definitions` module that we used as a reference point